test/object-to-string.test.js

```diff
@@ -0,0 +1,120 @@
+import convertObjectToString from '../lib/util/object-to-string';
+
+const cat1 = 'Pikachu_1';
+const cat2 = 'Raichu_2';
+const siv1 = 123;
+const siv2 = 214;
+const randomString = 'random string';
+const randomNumber = 42;
```

Comment:
Optional: It may be an idea to add Faker as a dev dependency and use that here

Reply:
Added. I was thinking about it too but it felt like an overkill for this little.